src/main/java/ReviewResponse2.java

```diff
@@ -0,0 +1,25 @@
+public class ReviewResponse2 {
+    private int line;
+    private String comment;
+
+    public ReviewResponse2(int line, String comment) {
+        this.line = line;
+        this.comment = comment;
+    }
+
+    public int getLine() {
+        return line;
+    }
+
+    public void setLine(int line) {
```

Comment:
Directly exposing a setter for 'line' attribute violates the Encapsulation principle. It would be better to have validation logic within the setter method. (Severity: Low)